tests/user/test_user_constants.py

```diff
@@ -198,6 +199,9 @@ def get_sample(sample_file: str) -> Union[str, bytes]:
 TEST_ADD_USER_SUCCESS_LOG = get_sample("add_user_success.log")
 TEST_ADD_USER_ERROR_LOG = get_sample("add_user_error.log")
 
+TEST_ADD_USER_ADDED_SECRET_SUCCESS_LOG = get_sample("add_user_added_secret_success.log")
+TEST_ADD_USER_ADDED_SECRET_ERROR_LOG = get_sample("add_user_added_secret_error.log")
```

Comment:
Can we use the naming convention "additional secret added" instead of "added secret" for these tests for clarity purposes?

Reply:
Agree, nice catch, I had missed that in the other name changes.